docs/events-interception.md

```diff
@@ -0,0 +1,190 @@
+    testplane.on(testplane.events.TEST_FAIL, (test) => {
+        // this handler will be called as usual,
+        // because intercepting the TEST_FAIL event does not change it in any way
+    });
+};
+```
+
+## Ignoring an event {#ignoring-event}
+
+To ignore an event and prevent it from propagating further, you need to return an empty object from the handler (in which the event is intercepted):
+
+```javascript
+module.exports = (testplane) => {
+    testplane.intercept(testplane.events.TEST_FAIL, ({ event, data }) => {
+        return {};
+    });
+
+    testplane.on(testplane.events.TEST_FAIL, (test) => {
+        // this handler will never be called
+    });
+};
+```
+
+## Delaying event processing {#delaying-events}
+
+The above approach with ignoring an event can be used to delay the occurrence of certain events, for example:
+
+```javascript
+module.exports = (testplane) => {
+    const intercepted = [];
+
+    testplane.intercept(testplane.events.TEST_FAIL, ({ event, data }) => {
+        // collect all TEST_FAIL events
+        intercepted.push({ event, data });
+
+        // and do not let them propagate further
+        return {};
+    });
+
+    testplane.on(testplane.events.END, () => {
+        // after the test run is finished, trigger all accumulated TEST_FAIL events
+        intercepted.forEach(({ event, data }) => testplane.emit(event, data));
+    });
+};
+```
+
+### Passing information between event handlers {#passing-information-between-event-handlers}
+
+Events that are triggered in the master process and in Testplane workers cannot exchange information via global variables.
+
+For example, this approach will not work:
+
+```javascript
+module.exports = (testplane) => {
+    let flag = false;
+
+    testplane.on(testplane.events.RUNNER_START, () => {
+        flag = true;
+    });
+
+    testplane.on(testplane.events.NEW_BROWSER, () => {
+        // false will be displayed because the NEW_BROWSER event
+        // is triggered in the testplane worker, and RUNNER_START in the master process
+        console.info(flag);
+    });
+
+    testplane.on(testplane.events.RUNNER_END, () => {
+        // true will be output
+        console.info(flag);
+    });
+};
+```
+
+But you can solve the issue like this:
+
+```javascript
+module.exports = (testplane, opts) => {
+    testplane.on(testplane.events.RUNNER_START, () => {
+        opts.flag = true;
+    });
+
+    testplane.on(testplane.events.NEW_BROWSER, () => {
+        // true will be output because the properties in the config,
+        // which have a primitive type (and the "opts" variable is part of the config),
+        // are automatically passed to workers during the RUNNER_START event
+        console.info(opts.flag);
+    });
+};
+```
+
+Or like this: see [example](./events/new_worker_process.md#usage) from the description of the [NEW_WORKER_PROCESS](./events/new_worker_process.md) event.
+
+### Parallel execution of plugin code {#parallel-execution-of-plugin-code}
```

Comment:
This too I believe would be a better fit in the main events article, not about intercepting them